Add documentation for GitHub App API changes within Terraform Cloud (#283)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/policy-sets.mdx

| `data.attributes.vcs-repo.oauth-token-id`     | string         |           | The OAuth Token ID Terraform Cloud should use to connect to the VCS host. This value must not be specified if `github-app-installation-id` is specified.                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |

> | `data.attributes.vcs-repo.github-app-installation-id` | string |           | The VCS Connection GitHub App Installation to use. This ID can be obtained from the user settings page. Requires previously authorizing the GitHub App and generating a user-to-server token. You can manage this from User Settings within Terraform Cloud. This value must not be specified if `oauth-token-id` is specified.                                                                                                                                                                                                               |

Reviewer: ```suggestion
| `data.attributes.vcs-repo.github-app-installation-id` | string |           | The VCS Connection GitHub App Installation to use. Find this ID on the user settings page. Requires previously authorizing the GitHub App and generating a user-to-server token. Manage the token from **User Settings** within Terraform Cloud. You can not specify this value if `oauth-token-id` is specified.                                                                                                                                                                                                               |
```
